docs/user-guide/at-tls-configuration.md

```diff
@@ -0,0 +1,32 @@
+# Configuring AT-TLS
+
+The communication server on z/OS provides functionality to encrypt HTTP communication for on-platform jobs. This functionality is referred to as Application Transparent Transport Layer Security (AT-TLS).
+
+## Configuration Parameters
+
+To enable AT-TLS for Zowe components, configure the following parameters:
+```yaml
+zowe:
+    network:
+        server:
+            attls: true
+```
+
+### High Availability Setup
+
+In a high availability setup where one LPAR (e.g., lpar1) is using AT-TLS and the second (e.g., lpar2) is also using AT-TLS, configure as follows:
```

Comment:
Just in case there is missing AT-TLS on any lpar?

Reply:
that's typo,thanks